tests/cypress/tests/e2e/admin-dashboard/admin-dashboard.feature

```diff
@@ -0,0 +1,24 @@
+Feature: Admin Dashboard Page E2E Testing
+
+    Scenario Outline: Admin User can archive a state report
```

Comment:
**Suggestion**

Since you aren't processing this test multiple times you can just use `Scenario` or `Rule`

`Scenario Outline` is only required for rerunning a test with different `Example` data https://cucumber.io/docs/gherkin/reference/#scenario-outline